crates/api/tests/traps.rs

```diff
@@ -114,9 +116,9 @@ fn test_trap_trace_cb() -> Result<(), String> {
     let trace = e.trace();
     assert_eq!(trace.len(), 2);
     assert_eq!(trace[0].module_name().unwrap(), "hello_mod");
-    assert_eq!(trace[0].func_index(), 1);
+    assert_eq!(trace[0].func_index(), 2);
```

Comment:
I'm curious as to why this changed. Are the indexes unstable such that we shouldn't be asserting on them?

Additionally, should we check the names here too (ideally, `run` -> `hello` or, in the future, `run` -> `hello` -> `throw`)?

Reply:
Forgot to address it in previous PR: I did not count imports, now it is fixed.